### ISSUE TITLE
[Bug] calling 911 with no  phone

### ISSUE THREAD

**Issue by Nussifer** [FrenzyRoleplay/server-issues#408]
**Describe the bug**
We have people been dropped in the water and still able to call 911, as EMS we can not tell who has and doesn't have phones and have been met with toxicity cause of it from gangs 

**To Reproduce**
1. get downed w/o phone
2.  do /911 [message]
3.  EMS/PD gets call

**Expected behavior**
get a message saying you don't have a phone

**Comment by cademullen** [comment]
@Nussifer Is this still an issue?

**Comment by Nussifer** [comment]
yes I had a man that I dumped in the water call 911 it's happened more than that time I have gone to 911 calls where people were dumped with nothing
